specification/liftrweightsandbiases/Liftr.WeightsAndBiases.Management/tspconfig.yaml

```diff
@@ -17,6 +17,7 @@ options:
     use-write-core: true
     namespace: "{package-dir}"
   "@azure-tools/typespec-python":
+    service-dir: "sdk/weightsandbiases"
```

Comment:
@ChenxiJiang333  Since all languages need service-dir, you could add it as global config like: https://github.com/Azure/azure-rest-api-specs/blob/15267537d1578b355fb1ef700873b64a12d32c6f/specification/contosowidgetmanager/Contoso.Management/tspconfig.yaml#L1-L3